apps/namadillo/src/atoms/integrations/services.ts

```diff
@@ -338,16 +338,20 @@ export const fetchIbcChannelFromRegistry = async (
   return getChannelFromIbcInfo(ibcChainName, channelInfo) || null;
 };
 
-export const simulateIbcTransferFee = async (
+export const simulateIbcTransferGas = async (
   stargateClient: SigningStargateClient,
   sourceAddress: string,
   transferMsg: MsgTransferEncodeObject,
   additionalPercentage: number = 0.05
 ): Promise<number> => {
-  const estimatedGas = await stargateClient.simulate(
-    sourceAddress!,
-    [transferMsg],
-    undefined
-  );
-  return estimatedGas * (1 + additionalPercentage);
+  try {
+    const estimatedGas = await stargateClient.simulate(
+      sourceAddress!,
+      [transferMsg],
+      undefined
+    );
+    return estimatedGas * (1 + additionalPercentage);
+  } catch (error) {
+    throw error;
```

Comment:
I believe this is not necessary anymore

Reply:
I would not risk :P I'll leave that for now until we have everything related to gas estimate 100% fixed